gulp-tasks/sass.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file
+ * This file contains function for sass gulp task
+ *
+ * The gulp task compiles and minifies scss/civicase.scss file into css/civicase.min.css.
+ * Also prefix the output css selector with `#bootstrap-theme` selector except the output.
```

Comment:
All these details of the function should be documented at the top of `sassTask` function, instead of at the file level.